x-pack/plugins/monitoring/server/telemetry_collection/get_all_stats.ts

```diff
@@ -28,9 +25,12 @@ export interface CustomContext {
  */
 export const getAllStats: StatsGetter<CustomContext> = async (
   clustersDetails,
-  { callCluster, start, end, esClient, soClient },
+  { callCluster, timestamp },
   { maxBucketSize }
 ) => {
+  const start = moment(timestamp).subtract(20, 'minutes').toISOString();
```

Comment:
Another constant used directly. Again, I would prefer if we had the time in a `telemetry_constants.ts` file.